### PR TITLE
Change favorite colors in dark mode so that the text is readable.

### PR DESCRIPTION
**Pull request by kaylanx**
Fix for https://github.com/sequelpro/sequelpro/issues/3635 I took willoller's approach.

### CODE REVIEW

**Comment by kaylanx** [comment]
dark-mode
light-mode

File: Source/SPFavoriteColorSupport.m [hunk 1]
```diff
@@ -80,6 +80,18 @@ - (NSColor *)colorForIndex:(NSInteger)colorIndex
 
 - (NSArray *)userColorList
 {
+	if (@available(macOS 10.13, *)) {
```
Comment (Kaspik) [inline Source/SPFavoriteColorSupport.m:83]: There should be macro for these checks

Comment (kaylanx) [reply Source/SPFavoriteColorSupport.m:83]: Ah right, you mean the stuff defined in SPCompatibility.h ? I missed that before

**Comment by Kaspik** [comment]
Check https://github.com/sequelpro/sequelpro/issues/3705

**Pull request closed without merging**